plugin/etcd/msg/path_test.go

```diff
@@ -10,3 +10,15 @@ func TestPath(t *testing.T) {
 		}
 	}
 }
+
+func TestPath(t *testing.T) {
```

Comment:
A function with the same name is declared 10 lines earlier in the same file. Did you test this?